### DISCUSSION TITLE
Access through wireguard

### DISCUSSION THREAD

**Discussion by JacobDK8200** [haugene/docker-transmission-openvpn#2230] [Q&A]
I am trying to connect to the transmission web interface remotely through wireguard.
I can access the server, other docker images, SMB and more. But not anything inside the transmission docker. 
I have no problem connecting when on my local network. 
Any suggestions are very welcome, thanks.
Here is my docker compose:

version: '2.1'
services:
 transmission-openvpn:
    volumes:
        - /srv/dev-disk-by-uuid-xx/appdata/transmission:/config
        - /srv/dev-disk-by-uuid-xx/SSD/downloads:/downloads
    environment:
        - PUID=1000
        - PGID=100
        - CREATE_TUN_DEVICE=true
        - OPENVPN_PROVIDER=NORDVPN
        # - OPENVPN_CONFIG=default
        - NORDVPN_COUNTRY=DK
        - NORDVPN_CATEGORY=legacy_p2p
        - NORDVPN_PROTOCOL=udp
        - OPENVPN_USERNAME=xx
        - OPENVPN_PASSWORD=xx
        - OPENVPN_OPTS=--inactive 3600 --ping 10 --ping-exit 60
        - WEBPROXY_ENABLED=false
        - LOCAL_NETWORK=192.168.1.0/24
        - TRANSMISSION_SCRAPE_PAUSED_TORRENTS_ENABLED=false
        - TRANSMISSION_INCOMPLETE_DIR=/downloads/
        - TRANSMISSION_WATCH_DIR=/config/transmission-home/watch
        - TRANSMISSION_DOWNLOAD_DIR=/downloads
        - TRANSMISSION_HOME=/config/transmission-home
        - TRANSMISSION_UTP_ENABLED=false
        - TRANSMISSION_WEB_UI=flood-for-transmission
     
        - DNS=8.8.8.8,8.8.4.4
    cap_add:
        - NET_ADMIN
    logging:
        driver: json-file
        options:
            max-size: 10m
    ports:
        - 9091:9091
        - 83:80
        - 6789:6789
        - 5076:5076
  
    restart: unless-stopped
    image: haugene/transmission-openvpn

 librespeed:
    image: linuxserver/librespeed
    container_name: librespeed
    network_mode: service:transmission-openvpn
    environment:
      - PUID=1000
      - PGID=100
      - TZ=Europe/Copenhagen
  
    volumes:
       - /srv/dev-disk-by-uuid-xx/appdata/librespeed:/config
    restart: unless-stopped

 nzbget:
    image: linuxserver/nzbget
    container_name: nzbget
    network_mode: service:transmission-openvpn
    environment:
      - PUID=1000
      - PGID=100
      - TZ=Europe/Copenhagen
    volumes:
      - /srv/dev-disk-by-uuid-xx/appdata/nzbget:/config
      - /srv/dev-disk-by-uuid-xx/SSD/downloads:/downloads
    restart: unless-stopped

**Comment by edgd1er** [comment]
wild guess, using wireguard, You have a 10.0.0.0/8 CIDR, your compose is asking transmission to allow 192.168.1.0/24 ( LOCAL_NETWORK=192.168.1.0/24 )
you could add 10.0.0.0/8 route, though, I would check that this range does not match the NORDVPN VPN ip.

**Reply by AKDub** [reply]
Would this make access only possible through wireguard, or would local access still be possible? I am looking to do the same but ideally would need it to be both

**Reply by pkishino** [reply]
yes, add both ranges to LOCAL_NETWORK

**Comment by JacobDK8200** [comment]
Thanks I think that’s it. 
How would one check that the nordvpn ip is not in the same range?

**Reply by pkishino** [reply]
you can check what ip the container is assigned from nord and check which range it is in
